src/reactapp/src/api/cart/getCheckoutAgreements/modifier.js

```diff
@@ -0,0 +1,3 @@
+export default function restSetGuestPaymentMethodModifier(result) {
+  return result;
```

Comment:
It would be better we modify the result here. I prefer converting array result into an object. we can use `reduce` function for this. Converting into an object makes referring to the item so easy. Else each time we need to iterate over the array and filter out the item. Also prettifying the result also good. The following format would be good
```
{
    [agreementID]: {
        id
        label
        content
        isHtml
    }
```